CHANGELOG.md

```diff
@@ -2,6 +2,22 @@
 
 All notable changes to the "Azure Api Management VS Code" extension will be documented in this file.
 
+## 1.2.0
+
+### New features
+
+- **MCP servers support**: This feature allows you view MCP servers in APIM and modify policies for the MCP servers.
```

Comment:
This feature enables you to view MCP servers within an APIM service and modify the policies associated with these MCP servers.